2015/presentations/with modern_peripherals: Python and Flask Piotr Dyba/text.md

```diff
@@ -0,0 +1,208 @@
+choosing the camera's focus point. 
+Passive applications include performance analysis of design, 
+layout and advertising. The main usage that eye tracking for web developers 
+is analysis of the web app that will improve ads, layout and design.
+Controlling the site with eyes will be the next step. It will be especial useful 
+for handicapped people with even severe body or mind damage. Think about 
+Steven Hawking, and how well he is doing despite his condition, but using eye 
+tracking in such a manner needs some level of focus. I think that when eye 
+trackers will get more popular, or using the built-in cameras for this purpose 
+will be more efficient and common… when we will get used to the technology,
+its advantages and some of people will start using it on daily basis, the eye 
+following ads will emerge. Closed only by blinking three times or roll eyes gestures
+
+#### Wrapper
+
+The Python wrapper is straight forward and one file only. 
+[Eye Tribe wrapper](https://github.com/baekgaard/peyetribe)
+It connects directly to the eye tribe daemon. The Eye Tribe was the easiest 
+device to use with python. The wrapper provides only information about point of gaze and
+ eyes location for each eye separately, but that is all what we need.
+
+#### JS vs Python
+I've compared efficiency of python solution against pure javascript one, during
+two tests, the first app types one for each approach was displaying live data and the second one was only collecting data. The main advantage of pure JS solution was ability to use it live without major glitches. The python solution still relied on JS to display the current 
+location of gaze so it was at the disadvantage from the start the situation
+was quite opposite when I used python only to gather the data and JS to display 
+the outcome overlaid on the survived website. So depending on the 
+application there we should choose the best solution.
+
+#### Hypothetical app
+Imaging an application that would measures and rate user’s art knowledge 
+by the way he looked at a picture or painting. What did he focus 
+on, and how long was he really looking at it. Wouldn't it be nice to be able
+to grade the ones photographic skill not only by how he takes photos but also
+how he looks at them.
+
+##Myo Armband
+ 
+The Myo Armband was introduced as a new way to interact with a  computer, but now it is 
+advertised more like presentation tool which it really excels in, so I often use it
+when presenting. If you desire stealth it is possible to go thorough the whole 
+presentation without audience knowing how the slides are being changed.
+The armband reads your muscle activity using electromyography a 
+sensor to read electric impulses going through the muscles, it also have 
+accelerometer so you can control your software with gestures and motion. 
+The first possible use that comes to my mind is to 
+simply to add control over the app to Myo it would be 
```

Comment:
Something wrong here: "to simply to..."? I'm not sure if I understand the use case correctly. It's about passing a control to Myo in case the app doesn't need text inputs. It is because it's not possible to use Myo to enter text, is that right?